share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,32 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "piratetalk",
+
+            // Customize these properties
+            name: "Talk like a Pirate!",
```

Comment:
> Name field is the one which gets displayed as the header in instant answer tab.

Yes, the `name` should be one of our predefined topics, or `Answer` when it's appropriate. In this case `Anwer` is the best tab name.